labelbox/schema/data_row.py

```diff
@@ -75,6 +73,8 @@ def update(self, **kwargs):
         # Convert row data to string if it is an object
         # All other updates pass through
         row_data = kwargs.get("row_data")
+        if not row_data and not kwargs.get("external_id"):
```

Comment:
The validation in ADV enforces that if `row_data` or `global_key` or `external_id` is set, it must not be an empty string.

What you have above is likely too strict, e.g. a user should be able to only update the `global_key`.

Reply:
Thanks, fixed